run_vert.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# VERT Docker Setup Script
+# This script clones VERT, builds a Docker image, and runs it.
+
+# Configuration
+REPO_URL="https://github.com/VERT-sh/VERT"
+IMAGE_NAME="vert-sh/vert"
+CONTAINER_NAME="vert"
+HOST_PORT=3000
+CONTAINER_PORT=80
+
+# Build-time environment variables
+PUB_ENV="production"
+PUB_HOSTNAME="vert.sh"
+PUB_PLAUSIBLE_URL="https://plausible.example.com"
```

Comment:
```suggestion
PUB_PLAUSIBLE_URL=""
```
this should be empty by default